src/qforte/utils/qubit_tapering.py

```diff
@@ -7,7 +7,7 @@
 from itertools import product
 from copy import deepcopy
 
-def find_Z2_symmetries(hamiltonian, taper_from_least = False, debug = False):
+def find_Z2_symmetries(hamiltonian, taper_from_least = True, debug = False):
```

Comment:
Did you mean to change this flag?

Reply:
I did. Based on a few UCCSD runs (using my local version of QForte in which qubit tapering is interfaced with UCCNVQE), this option leads to quantum circuits with less CNOT gates (of course, this could always be a coincidence).